src/test/ui/closures/2229_closure_analysis/migrations/closure-body-macro-fragment.stderr

```diff
@@ -0,0 +1,37 @@
+warning: changes to closure capture in Rust 2021 will affect drop order
+  --> $DIR/closure-body-macro-fragment.rs:8:17
+   |
+LL |           let f = || $body;
+   |  _________________^
+LL | |
+LL | |         f();
+LL | |     }};
+   | |     - in Rust 2018, `a` is dropped here, but in Rust 2021, only `a.0` will be dropped here as part of the closure
+LL | |     ($body:block) => {{
+LL | |         m!(@ $body);
+   | |__________________^
```

Comment:
That's... an unfortunate span.

Reply:
Yeah, some weird things are  going on when mixing `:block` and `:expr` fragments. And closures just use their first and last token to create their full span, which isn't always great. Something to improve another time. ^^

Reply:
Oh aboslutely, mixed spans involving macros have long standing issues :)